### PR TITLE
Improve radio fieldset

### PR DESCRIPTION
**Pull request by guiburato**
- implement new optional param(radioHelpText) for radio input

### CODE REVIEW

File: src/main/resources/templates/fragments/inputs/radio.html [hunk 1]
```diff
@@ -19,5 +20,9 @@
             aria-invalid=${hasError},
             data-follow-up=${followUpId}">
     <span th:text="${label}"></span>
+    <p th:if="${hasHelpText}"
+       th:id="${inputName} + '-' + ${value} + '-help-text'"
+       th:utext="${radioHelpText}"
+       class="text--help" style="margin-bottom: 0px"></p>
```
Comment (sree-cfa) [inline src/main/resources/templates/fragments/inputs/radio.html:26]: I think this could be `class="text--help with-no-padding"`

Comment (sree-cfa) [inline src/main/resources/templates/fragments/inputs/radio.html:26]:
```suggestion
       class="text--help with-no-padding"></p>
```

Comment (guiburato) [reply src/main/resources/templates/fragments/inputs/radio.html]: Nice! Thanks.

**Comment by bseeger** [comment]
Out of curiosity - the `radioFieldset` does have helptext - do you have a use case where you're not using a field set?  Or a use case where there is helptext for each radio button?

**Comment by sree-cfa** [comment]
@bseeger  checkboxInSet has this at the selection-level too. We want some additional text for a radio option
Screen Shot 2024-02-07 at 3 03 48 PM

**Review by bseeger — APPROVED** [review]
LGTM!

**Pull request merged**